test/unit/range/views/view_pseudo_random_access_test.cpp

```diff
@@ -0,0 +1,62 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/decorator/gap_decorator.hpp>
+#include <seqan3/range/views/pseudo_random_access.hpp>
+#include <seqan3/range/views/to_char.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+template <typename t>
+class pseudo_random_access_test : public ::testing::Test
```

Comment:
why don't you use the iterator test_template :D

Reply:
Since the class is not defining its own iterator but forwarding it, it seemed not so important. But i don’t mind adding it.